test/integration/config.go

```diff
@@ -9,5 +9,4 @@ import (
 var (
 	pathRoot   = tools.ProjectDir()
 	pathOutput = path.Join(pathRoot, "testoutput")
-	pathVarRun = path.Join(pathOutput, "run")
```

Comment:
why has this become a problem? Not sure if related, but I noticed a few of my PRs are failing a few tests as well

Reply:
Since we'll be removing the BPF file system dependency, I removed all the tracking support for it. Timing is different with the new BPF cleanup and I think we don't always clean it up on time in tests.